### ISSUE TITLE
Brave Not Opening On Garuda Linux (Arch Based) After Latest Update

### ISSUE THREAD

**Issue by Hamza-b93** [brave/brave-browser#29885]
## Description 
As the title suggests, the latest version of Brave Browser on Desktop (version 1:1.50.121-1) is failing to open at all after the latest update. Nothing appears in the process manager either. Running Brave from the CLi gives the following output:

`fish: Job 1, 'brave' terminated by signal SIGILL (Illegal instruction)`

## Steps to Reproduce

   1. Open CLI.
   2. Type brave
   3. Press Enter

## Actual result:
`(fish: Job 1, 'brave' terminated by signal SIGILL (Illegal instruction))`

## Expected result:
Brave Browser opens.

## Brave version (brave://version info)
Version: 1:1.50.121-1
Package: brave-bin (AUR)

- Can you reproduce this issue with the current release?
Yes.

- Can you reproduce this issue with the beta channel? 
Not checked.

- Can you reproduce this issue with the nightly channel? 
Yes. The following error is thrown:
`/usr/bin/brave-browser-nightly: line 48: 147034 Illegal instruction     (core dumped) "$HERE/brave" "$@"`

## Miscellaneous Information:
Operating System: Garuda Linux (Arch Based).
Source: AUR.
Processor: AMD Ryzen 7 3700-X.
Kernel: 6.2.11-zen1-1-zen.

**Comment by CRCinAU** [comment]
Somewhat related, brave-browser-1.50.121-1.x86_64 does not function on Fedora 37 either.

It does a coredump - but the fatal line in the journal seems to be:
```
kernel: traps: brave[4870] trap invalid opcode ip:56224c1cb90c sp:7ffe6aad7a00 error:0 in brave[562247dda000+aefe000]
```

This is using the official repo for Fedora hosted on s3.

**Comment by kjozwiak** [comment]
We pushed out `1.50.125 Chromium: 112.0.5615.165` which should fix the issue for everyone. If you were having issues, please let us know if the above release fixed your issue. You can also download the `deb` files manually via https://github.com/brave/brave-browser/releases/tag/v1.50.125. Just for some context, the above was fixed by upgrading chromium from `112.0.5615.138` to `112.0.5615.165`.

You can also find some context of what happened via https://github.com/brave/brave-browser/issues/29871#issuecomment-1516644603.
